examples/basic/src/components/Overlay.tsx

```diff
@@ -23,14 +21,15 @@ import {
   type VideoTrack,
   type AudioTrack,
 } from 'react-native-video';
-import {
-  toast,
-  Seeker,
-  AudioTrackSelector,
-  TextTrackSelector,
-  VideoTrackSelector,
-  TopControl,
-} from '../components';
+
+import {toast} from './Toast';
+import {Seeker} from './Seeker';
+import {AudioTrackSelector} from './AudioTracksSelector';
+import {VideoTrackSelector} from './VideoTracksSelector';
+import {TextTrackSelector} from './TextTracksSelector';
+import {TopControl} from './TopControl';
+import {ToggleControl} from './ToggleControl';
+import {MultiValueControl} from './MultiValueControl';
```

Comment:
I think it's better to import as shown below:
```
import {
  toast,
  Seeker,
  AudioTrackSelector,
  TextTrackSelector,
  VideoTrackSelector,
  TopControl,
  ToggleControl,
   MultiValueControl
} from '../components';
```

Reply:
But will produce the circular dependency issue.. as ../component/index includes Overlay, Overlay will try to include itself...

Reply:
But, maybe overlay shouldn't be in component?!

Reply:
You're right. I approved it.